emily/handler/src/database/entries/mod.rs

```diff
@@ -60,6 +64,50 @@ pub mod deposit;
 /// Withdrawal table entries.
 pub mod withdrawal;
 
+// Event structure
+// -----------------------------------------------------------------------------
+
+/// Status entry.
+#[derive(Clone, Default, Debug, Eq, PartialEq, PartialOrd, Ord, Serialize, Deserialize, Hash)]
+#[serde(rename_all = "PascalCase")]
+pub enum StatusEntry {
+    /// Transaction hasn't yet been addressed by the sBTC Signers.
+    #[default]
+    Pending,
+    /// Transaction was dealt with by the signers at one point but is now being
+    /// reevaluated. The Signers are aware of the operation request.
+    Reevaluating,
```

Comment:
I would change this to reprocessing. Re-evaluating sounds like the signers are deciding again whether to act on the request, which I don't think they do.

Reply:
Done!